### ISSUE TITLE
Button label in Cal-ITP Administrator doesn't meet accessibility standards

### ISSUE THREAD

**Issue by indexing** [cal-itp/benefits#2867]
## To Reproduce

1. Go to https://test-benefits.calitp.org/admin/. Login as `cst-user`.
2. Click on `New enrollment` to begin an in-person enrollment.
3. Complete enrollment.
4. Note the `Done` button label does not pass contrast minimums. It seems to be in a state other than active.

## Expected behavior

The `Done` button label passes contrast minimums.

## Screenshots

![Image](https://github.com/user-attachments/assets/0ddfdc3d-0192-40b4-af8b-dc692283bf46)

## Desktop (please complete the following information)

- OS: MacOS
- Browser: Chromium (Arc)
- Version 1.91.2
